crispy_bulma/layout.py

```diff
@@ -227,6 +228,7 @@ def render(
             template=template,
             template_pack=template_pack,
             extra_context=extra_context,
+            attrs=self.attrs
```

Comment:
a comma at the end is missing

Reply:
Sorry for that messy PR... I shouldn't do that directly on github website when tired...
Anyway, that's working for me.
Thanks for your work.

Reply:
@davy39 your fix is available on pypi now - new release 0.8.1, thanks again!